draft-wang-ppm-dap-taskprov.md

```diff
@@ -47,79 +47,85 @@ allows the task configuration to be provisioned in-band.
 
 # Introduction
 
-The DAP protocol {{?DAP=I-D.draft-ietf-ppm-dap-01}} enables secure aggregation
-of a set of reports submitted by clients. This process is centered around a
+The DAP protocol {{!DAP=I-D.draft-ietf-ppm-dap-01}} enables secure aggregation
```

Comment:
question: what's the difference between ! and ? in this format? is there a standard doc I can refer to for this kind of formatting?

Reply:
"!" denotes "normative" reference; "?" denotes 'informative" reference. If a reference is "normative", then it is effectively part of the spec.